lib/Runtime/Library/TypedArray.cpp

```diff
@@ -2844,7 +2844,7 @@ namespace Js
         return false;
     }
 
-    BOOL TypedArrayBase::ValidateIndexAndDirectSetItem(__in Js::Var index, __in Js::Var value, __in bool * isNumericIndex)
+    BOOL TypedArrayBase::ValidateIndexAndDirectSetItem(__in Js::Var index, __in Js::Var value, __out bool * isNumericIndex)
```

Comment:
Sounds like a good idea - but AFAICT we haven't done that anywhere in our code base so far. Wonder why?